wormhole-connect/src/utils/wallet/index.ts

```diff
@@ -7,14 +7,15 @@ import {
   WormholeContext,
   SendResult,
 } from '@wormhole-foundation/wormhole-connect-sdk';
-import { CHAIN_ID_SEI, postVaaSolanaWithRetry } from '@certusone/wormhole-sdk';
+import { postVaaSolanaWithRetry } from '@certusone/wormhole-sdk';
 import { ContractReceipt } from 'ethers';
 import { NotSupported, Wallet } from '@xlabs-libs/wallet-aggregator-core';
 import {
   EVMWallet,
   MetamaskWallet,
   WalletConnectWallet,
 } from '@xlabs-libs/wallet-aggregator-evm';
+import { SeiTransaction } from '@xlabs-libs/wallet-aggregator-sei';
```

Comment:
I still have to import this type from `wallet-aggregator-sei`, but it properly tree shakes and avoids bloating the main bundle size.